src/cinder/ObjLoader.cpp

```diff
@@ -402,7 +402,7 @@ void ObjLoader::loadGroupNormalsTextures( const Group &group, map<VertexTriple,i
     bool hasColors = mMaterials.size() > 0;
 	for( size_t f = 0; f < group.mFaces.size(); ++f ) {
 		vec3 inferredNormal;
-		bool forceUnique = false;
+		bool forceUnique = !mOptimizeVertices;
```

Comment:
Space goes between `!` and variables, in the two cases below as well (I'm going to add this to the CONTRIBUTING.md guide, since Andrew keeps reminding us that this is the preferred way).